style.css

```diff
@@ -695,3 +719,166 @@ contact me section
     width: 159px;
   }
 }
+
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 920px;
+}
```

Comment:
- [ ] As requested by the previous reviewer, Please make the border of the model a little rounded to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2702&t=rPMLdWrQ30by8pU6-0) design.
  - You should use the `border-radius:` property on this class to make the model corners rounded.

### Your design

![image](https://user-images.githubusercontent.com/83803414/209107598-decede72-6765-4f1b-ae11-8dd7975ba35c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/209107560-4baf5a0a-7977-446a-838d-4e1a3c8305bd.png)